_notices/rsn0023.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 23 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Support for Python 3.10 in v23.02"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v22.12+"
+notice_created: 2022-11-21
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-11-21
+---
+
+## Overview
+
+With the depracation of `Python 3.9` announced in [RSN 22](/notices/rsn0022), development
+effort has been redirected to support `Python 3.10` in `v23.02` nightly releases.
+
+## Status
+
+- `Python 3.10` nightly support started on 16-Nov-022 for both `conda` and
```

Comment:
```suggestion
- `Python 3.10` nightly support started on 16-Nov-2022 for both `conda` and
```

Reply:
Is this true?

I don't see nightly Docker images for Python 3.10. https://hub.docker.com/r/rapidsai/rapidsai-nightly/tags?page=1&name=3.10

I don't see nightly conda packages for Python 3.10. https://anaconda.org/rapidsai-nightly/cudf/files

We haven't updated packaging info to support Python 3.10, either.